### ISSUE TITLE
Flyctl install error: 'Unable to find latest Flyctl release on GitHub'

### ISSUE THREAD

**Issue by gregmsanderson** [superfly/flyctl#272]
Currently CI deploys are failing because it tries to install fly on a blank image, and can't. I tried it locally and found the same:

```
~ $ curl -L https://fly.io/install.sh | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2208    0  2208    0     0   1718      0 --:--:--  0:00:01 --:--:--  1790
Error: Unable to find latest Flyctl release on GitHub.
```

Issue with github? Strange!

**Comment by michaeldwan** [comment]
@gregmsanderson I'm getting the same thing from a new alpine image, we'll dig in!

**Comment by codepope** [comment]
This has been fixed with the release of 0.0.138 and there'll be a new ticket to address the underlying cause.
